src/_content/posts/volunteer-at-djcus.md

```diff
@@ -0,0 +1,110 @@
+
+### **Network Like Never Before**
+
+Volunteers have unique opportunities to connect with attendees, speakers, and Django core contributors. You'll naturally meet people throughout the event, building relationships that can last well beyond the conference.
+
+### **Develop New Skills**
+
+Whether it's improving your communication skills, learning event logistics, or gaining experience in customer service, volunteering offers practical skill development you can't get anywhere else.
+
+### **Be Part of the Magic**
+
+There's something special about being part of the team that makes DjangoCon happen. You'll experience the satisfaction of seeing your efforts directly contribute to creating memorable moments for hundreds of Django developers.
+
+## 
+
+## **What We're Looking For**
+
+We need enthusiastic volunteers who are:
+
+* Already registered for DjangoCon US 2025 with a valid ticket  
+* Passionate about the Django community  
+* Reliable and committed to their volunteer shifts  
+* Friendly and welcoming to all attendees  
+* Available for specific time slots during the event
+
+## **Volunteer Opportunities**
+
+We have various volunteer roles available across different areas:
+
+**Registration & Check-in**
+
+* Welcome attendees and help with badge pickup  
+* Assist with registration troubleshooting  
+* Direct people to sessions and facilities
+
+**Session Support**
+
+* Assist speakers with microphones and presentation setup  
+* Monitor room capacity and help with seating  
+* Introduce Speaker  
+* (Questions to attendees)
+
+**Online Talks  Coordination**
+
+* You can help with the technical stuff that always goes wrong during virtual events. This means jumping in when someone's audio cuts out, helping speakers figure out screen sharing, or guiding confused attendees through the conference platform. It's basically being the friendly tech support person everyone needs.  
+* Address questions from the online attendees to the speaker 
+
+
+## **Time Commitment**
+
+We understand that as attendees, you want to experience the conference too\! Volunteer shifts are designed to be flexible:
+
+* Most shifts are 2-4 hours long  
+* You can choose shifts that work around the talks you want to attend  
+* Many roles allow you to still listen to presentations while helping   
+* Sprint volunteers can participate in sprints while supporting the logistics
+
+## 
+
+## **Ready to Get Involved?**
+
+### **Apply to Volunteer**
+
+Ready to join our volunteer team? Submit your application here: **[https://docs.google.com/spreadsheets/d/1H4X4X7DtWOKmGe\_S0KSZjSuXRjqcps4OUbXzNir2u1U/edit?gid=1846864033\#gid=1846864033](https://docs.google.com/spreadsheets/d/1H4X4X7DtWOKmGe_S0KSZjSuXRjqcps4OUbXzNir2u1U/edit?gid=1846864033#gid=1846864033)** 
```

Comment:
Somehow the first link in brackets has an extra character that makes it invalid. Below both are correct: 

**[https://docs.google.com/spreadsheets/d/1H4X4X7DtWOKmGe_S0KSZjSuXRjqcps4OUbXzNir2u1U/edit?gid=2088259231#gid=2088259231](https://docs.google.com/spreadsheets/d/1H4X4X7DtWOKmGe_S0KSZjSuXRjqcps4OUbXzNir2u1U/edit?gid=2088259231#gid=2088259231)**